src/translation/german.c

```diff
@@ -104,6 +104,7 @@ static translation_string all_strings[] = {
     {TR_CONFIG_ROAMERS_DONT_SKIP_CORNERS, "Wandernde Arbeiter kürzen an Ecken nicht ab"},
     {TR_CONFIG_SHOW_MAX_POSSIBLE_PROSPERITY, "Anzeige des maximalen Wohlstands mit aktuellen Behausungen."},
     {TR_CONFIG_AUTO_KILL_ANIMALS, "Einwohner werden automatisch harmlose Tiere töten"},
+	{TR_CONFIG_DRAW_CLOUD_SHADOWS, "Wolkenschatten"},
```

Comment:
Just check the spacing on this one. I suspect your editor is set to a different number of 'spaces per tab'.